libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/QnAMaker/QnAMakerDialog2.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.AI.QnA;
+using Microsoft.Bot.Builder.AI.QnA.Dialogs;
+using Microsoft.Bot.Expressions;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.QnA
+{
+    public class QnAMakerDialog2 : QnAMakerDialog
```

Comment:
Could we consider another name like QnAMakerDialogDeclarative or QnaMakerDeclarativeDialog?

Reply:
This will merge with QnAMAkerDialog when we GA adaptive/lg etc.

---
In reply to: [352373855](https://github.com/microsoft/botbuilder-dotnet/pull/3048#discussion_r352373855) [](ancestors = 352373855)